sample-notificator-domain/src/main/groovy/life/qbic/business/subscription/fetch/FetchSubscriber.groovy

```diff
@@ -0,0 +1,42 @@
+package life.qbic.business.subscription.fetch
+
+import life.qbic.business.exception.DatabaseQueryException
+import life.qbic.business.subscription.Subscriber
+import life.qbic.datamodel.samples.Status
+
+import java.time.LocalDate
+
+/**
+ * <h1>Use case for fetching subscriber information from the database</h1>
+ *
+ * <p>Fetches the subscribers for whos subscriptions had updates on e.g a given date</p>
+ *
+ * @since 1.0.0
+ *
+*/
+class FetchSubscriber implements FetchSubscriberInput{
+    private final FetchSubscriberDataSource ds
+    private final FetchSubscriberOutput output
+
+    FetchSubscriber(FetchSubscriberDataSource ds, FetchSubscriberOutput output){
+        this.ds = ds
+        this.output = output
+    }
+
+    @Override
```

Comment:
This is not necesarry since JavaDoc is automatically copied if not provided for overwritten methods https://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#inheritingcomments